xcodeproj/internal/bazel_integration_files/calculate_output_groups.py

```diff
@@ -109,19 +127,27 @@ def _calculate_label_and_target_ids(
 
 def _calculate_guid_labels_and_target_ids(base_objroot):
     pif_cache = f"{base_objroot}/XCBuildData/PIFCache"
+    project_cache = f"{pif_cache}/project"
+    target_cache = f"{pif_cache}/target"
 
-    # For the first build, the PIFCache might not exist yet, so we wait a bit=
-    # for it to exist
-    wait_counter = 0
-    time_to_wait = 10
-    while not os.path.exists(pif_cache):
-        time.sleep(1)
-        wait_counter += 1
-        if wait_counter > time_to_wait:
-            break
+    # The PIF cache will only be created before the `SetSessionUserInfo`
+    # command, which normally happens when a project is opened. If Derived Data
+    # is cleared  while the project is open
+    if not (os.path.exists(project_cache) and os.path.exists(target_cache)):
+        print(
+            f"""\
+error: PIFCache ({pif_cache}) doesn't exist. If you manually cleared Derived \
```

Comment:
I got this error now. If it's not critical, can we make this a warning instead?

Reply:
It is critical, as you will build all the targets in the scheme, which could make the build take a lot longer (as it could build a bunch of tests). This is a quick fail, and an easy fix.